stories/Icons.stories.tsx

```diff
@@ -10,7 +11,7 @@ export const Icons: React.FC = () => (
     {sortName.map((value, index) => {
       return (
         <div key={index} className="preview">
-          <Icon iconName={value}></Icon>
+          <Icon iconName={value as TkIcon}></Icon>
```

Comment:
Let's avoid "as Type" as much as we can

Reply:
The name of the Icons are coming from `tk-icons.codepoints.json` and they are parsed as a string[]. T. See: https://github.com/SymphonyPlatformSolutions/symphony-bdk-ui-toolkit-components/blob/b79b161433f64527e511b7e001af84a1f742c5b0/stories/Icons.stories.tsx#L6
They will be the same as the types on the tk-icons.ts file but typescript can not know when is parsing the file.